scripts/test.sh

```diff
@@ -226,6 +226,8 @@ openssl*)
 		DEPS=""
 		use_custom_openssl $1
 	elif test "$TEST" = "openssl-awslc"; then
+		CONFIG="${CONFIG} --enable-drbg"
+		TESTS_PLUGINS="${TESTS_PLUGINS} drbg"
```

Comment:
We'll need this for OpenSSL too at some point, so you could also add these to the constants directly.